src/helpers/translation.ts

```diff
@@ -1392,3 +1394,96 @@ export const codeMessages = defineMessages({
     defaultMessage: 'payment proceed is error',
   },
 })
+
+export const questionLibraryMessage = {
+  ui: defineMessages({
+    sortQuestions: { id: 'questionLibrary.label.sortQuestions', defaultMessage: '排序題目' },
+    addQuestion: { id: 'questionLibrary.label.addQuestion', defaultMessage: '新增題目' },
+    useZhuYinFont: { id: 'questionLibrary.label.useZhuYinFont', defaultMessage: '使用注音字形' },
+    isAnswer: { id: 'questionLibrary.label.isAnswer', defaultMessage: '此為正確解答' },
+    addOption: { id: 'questionLibrary.label.addOption', defaultMessage: '新增選項' },
```

Comment:
this naming convention might be `questionLibrary.label.xxxx` ?

Reply:
sorry 
Because I move these translations from label to ui, and I forgot to rename id
I modified in below commit
thx